### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by slackmcgrath**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,4 +11,4 @@ Testing as assignee
 
 adding as reviewer
 
-19
+20!
```
Comment (MikaelaMcGrath) [inline README.md:14]:
```suggestion
21
```

**Comment by MikaelaMcGrath** [comment]
Hey

File: README.md (hunk 1 above)
Comment (slackmcgrath) [reply README.md:14]: I am in this thread

Comment (MikaelaMcGrath) [reply README.md:14]: you sure?

Comment (MikaelaMcGrath) [reply README.md:14]: @slackmcgrath work smarter

**Pull request merged**